test/libsolidity/syntaxTests/experimental/parsing/forall_free_function_no_parentheses_single_type_var.sol

```diff
@@ -0,0 +1,34 @@
+pragma experimental solidity;
+
+// The parser currently allows unary parameter lists without parentheses
```

Comment:
And it has a fat TODO about that - I'd not change that for the time being, but is there really any merit in adding tests for this stuff at this point :-D? But yeah, just saying - fine with keeping it here :-).

Reply:
Well, this was initially a negative test checking that this generates an error and, to my surprise, it didn't - but confusingly only for a single argument. Then I found the TODO :)

I guess you have a point. Maybe I'm better off removing it.

Reply:
Tests removed.